front/app/containers/Admin/messaging/texting/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { List as TextsList } from 'components/admin/ResourceList';
+
+class TextingCampaigns extends React.PureComponent {
+  render() {
+    return (
+      <>
+        <TextsList>
+          <p>Texting main page placeholder</p>
+        </TextsList>
+      </>
+    );
```

Comment:
1) New components should be functional components.
2) The `<></>` wrapper is not needed.

Reply:
This one is just a placeholder for now, in my next PR it will be completely removed and updated to functional.